### PR TITLE
isPublic -> isActive

### PR DESCRIPTION
**Pull request by xdamman**

### CODE REVIEW

File: server/controllers/groups.js [hunk 1]
```diff
@@ -311,10 +311,13 @@ export const create = (req, res, next) => {
     .tap(g => {
       // if there is a host id provided, we add the collective to the host
       if (group.HostId) {
-        return User.findOne({ where: { id: group.HostId }}).tap(host => _addUserToGroup(g, host, {role: roles.HOST, remoteUser: creator}))
+        return User.findOne({ where: { id: group.HostId }}).tap(h => {
+          host = h;
```
Comment (asood123) [inline server/controllers/groups.js]: why do we need to store `host` separately?

**Comment by xdamman** [comment]
Right now I did it on the front end. See pull request there.

File: server/controllers/groups.js (hunk 1 above)
Comment (xdamman) [inline server/controllers/groups.js]: We pass it to the activity data. I just reused the same pattern used in that segment for groupCreated. I think it's clearer and easier to read/follow.

Comment (asood123) [inline server/controllers/groups.js]: 👍

**Pull request merged**